cfgov/templates/wagtailadmin/css/form-explainer.css

```diff
@@ -0,0 +1,19 @@
+.explainer_notes .fields li:first-child label {
+    width: 100% !important;
```

Comment:
It would be good to add some code comments for the `!important`s in this file of why they are necessary. What are they overriding?

Reply:
Thanks for flagging! I've updated selectors in 7bb341d842aee1336c1bb51de736bbeaa8dd8437 to manage the overrides.